rusoto/services/acm/README.md

```diff
@@ -0,0 +1,43 @@
+
+# Rusoto Acm
+Rust SDK for AWS Certificate Manager
+
+You may be looking for:
+
+* [An overview of Rusoto][rusoto-overview]
+* [AWS services supported by Rusoto][supported-aws-services]
+* [API documentation][api-documentation]
+* [Getting help with Rusoto][rusoto-help]
+
+## Requirements
+
+Rust 1.15.0 or later is required.
```

Comment:
Any thoughts on dropping this required Rust version from the README for each service?  It'd be yet another place for us to try to keep in sync.  Perhaps something like I did in the CONTRIBUTING file where it asks you to check TravisCI and Appveyor configs for what versions we support outside of `stable`: https://github.com/rusoto/rusoto/pull/601/files .